recipes/uppasd/bld.bat

```diff
@@ -0,0 +1,19 @@
+setlocal EnableDelayedExpansion
+
+md build
+if errorlevel 1 exit 1
+
+
+:: LDFLAGS defaults to -Wl,-defaultlib:D:/bld/uppasd_1763370055533/_build_env/lib/clang/19/lib/windows/clang_rt.builtins-x86_64.lib
+:: but the syntax -Wl,... is not correct for flang
+set "LDFLAGS="
```

Comment:
This works on many other feedstocks, what problems did you encounter?

Reply:
Flang outputs a warning about an unknown argument: https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=1392090&view=logs&j=53c3d5f4-e7c7-5b53-4b94-211701b2ba17&t=8f042650-8479-5761-7a4a-8d021c0bb2de&l=2734

I have disabled overwriting the flag in 22417bf. I assume it will be build successfully because the flag is just ignored.

Reply:
The warning has disappeared in the pipeline run: https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=1394931&view=logs&j=53c3d5f4-e7c7-5b53-4b94-211701b2ba17&t=8f042650-8479-5761-7a4a-8d021c0bb2de Maybe it was some sort of false-positive triggered by one of the other warnings.

I'll remove the flag from bld.bat entirely.